app/models/manageiq/providers/datawarehouse_manager.rb

```diff
@@ -1,5 +1,8 @@
 module ManageIQ::Providers
   class DatawarehouseManager < BaseManager
+    has_many :datawarehouse_nodes, :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
As above, this relationship is different then that of openshift and kubernetes; hawkular/datawarehouse_manager will never have these. Would it make sense to add them only on the child manager? (cc @durandom )